lib/facility.rb

```diff
@@ -1,14 +1,42 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name, :address, :phone, :services, :facility, :registered_vehicles, :collected_fees
 
-  def initialize(name, address, phone)
-    @name = name
-    @address = address
-    @phone = phone
+  def initialize(facility)
+    @name = facility[:name]
+    @address = facility[:address]
+    @phone = facility[:phone]
     @services = []
+    @facility = []
```

Comment:
we shouldn't need this attribute, nor will we need an attr_reader for it. I say this because I don't see anywhere in the interaction pattern where we call `.facility` on a `Facility` object.